src/utils/domUtils.ts

```diff
@@ -20,13 +20,13 @@ export module Utils {
       return bbox;
     }
 
-    export var POLYFILL_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
+    export var REQUEST_ANIMATION_FRAME_TIMEOUT_MILLISECONDS = 1000 / 60; // 60 fps
 
     export function requestAnimationFramePolyfill(fn: () => void) {
```

Comment:
>Which basically says that the word polyfill here means almost nothing.  Our `Map`s by this definition are I think polyfills and thus having the variable name `POLYFILL_TIMEOUT_MILLISECONDS` I think is really not useful

I don't see how the first part follows from the second part here.

For that matter, is there a reason this needs to be exported?